docs/process.rst

```diff
@@ -862,6 +862,9 @@ val(x)          x
 file(x)         (not supported)
 file('name')    'name'
 file(x:'name')  x:'name'
+path(x)         (not supported)
+path('name')    (not supported)
+path(x:'name')  (not supported) 
```

Comment:
Think, `path` does not support anymore this notation but it should be `path(x, stageAs:'name')`

Reply:
You are right, just tested and confirmed,

While `path(x)` and `path('name')` work as expected `path(x:'name')` has to be replaced by `path(x, stageAs:'name')`

```suggestion
path(x, stageAs:'name')  (not supported) 
```